packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -209,6 +209,8 @@ export const messages = {
   force_apex_log_get_no_logs_text: 'No Apex debug logs were found',
   force_apex_log_get_pick_log_text: 'Pick an Apex debug log to get',
   force_apex_log_list_text: 'Getting Apex debug logs',
+  force_apex_log_get_library: 'Apex Library: Get Logs',
```

Comment:
@hbc-bindu @ekapner What do we think about the message "Log Get Apex". This text will be displayed in the progress bar while the command is running: 
<img width="857" alt="Screen Shot 2020-07-17 at 12 41 38 PM" src="https://user-images.githubusercontent.com/21326913/87815559-6ca31f00-c82b-11ea-9e0e-c3fb45e398f7.png">

Reply:
We currently display the text used to run the CLI command behind the scenes:
<img width="658" alt="Screen Shot 2020-07-17 at 12 42 19 PM" src="https://user-images.githubusercontent.com/21326913/87815614-86dcfd00-c82b-11ea-8284-fd68253985fa.png">

Reply:
@AnanyaJha - what about? And is it logs, plural, or just ONE log? I noticed the command above says "Get Logs"
Starting Get Apex Log
Finished Get Apex Log 
If could be multiple logs, then I would just add an "s" to Log

Reply:
Sounds good to me, we can go with `Get Apex Logs`, thanks!

Reply:
Yeah I was thinking to change the name as well. Like "Log Get Apex" sounds little bit weird so lets go with "Get Apex Logs" 👍🏻

Reply:
Thank you!